test/metabase/integrations/slack_test.clj

```diff
@@ -236,3 +236,7 @@
               (is (= false (slack/valid-token? "abc")))
               (is (= {} (mt/summarize-multipart-email #"Your Slack connection stopped working.")))
               (is (slack/slack-token-valid?)))))))))
+
+(deftest slack-cache-updated-at-nil
```

Comment:
wdyt if we put this test in the `update-slack-settings-test` under `metabase.api.slack-test`?
I think it'll have more context there other than just checking if it has default value here.

Reply:
Good idea! I'll add it there as well.

Reply:
FYI, Slack test under `metabase.api.slack-test` is failing across all Java versions in CircleCI:
https://app.circleci.com/pipelines/github/metabase/metabase/32972/workflows/e625bb8c-e231-4eca-a0ff-acfd219b236a.

I reran failed jobs to see if that's going to fix it. But either way, this is something that @escherize should take a look at.